### PR TITLE
bugfix/155 - Accept entrail property in spawnPatternModel

### LINKED ISSUE #155: Accept entrail property in spawnPatternModel
Some spawnPatterns set an explicit `entrail` value that the `SpawnPatternModel` should accept.  This is value is currently defaulted and not allowing passed in values.

### PR DESCRIPTION
**Pull request by felixscheffer**
Resolves openscope/openscope#155

### CODE REVIEW

File: src/assets/scripts/client/trafficGenerator/SpawnPatternModel.js [hunk 1]
```diff
@@ -489,6 +489,7 @@ export default class SpawnPatternModel extends BaseModel {
         this.speed = this._extractSpeedFromJson(spawnPatternJson);
         this.method = spawnPatternJson.method;
         this.rate = parseFloat(spawnPatternJson.rate);
+        this.entrail = _get(spawnPatternJson, 'entrail', this.entrail);
```
Comment (n8rzz) [inline src/assets/scripts/client/trafficGenerator/SpawnPatternModel.js]: seems simple enough.  does this actually fix the `Surge` spawnPattern`s?

Comment (felixscheffer) [reply src/assets/scripts/client/trafficGenerator/SpawnPatternModel.js]: The surge spawnPatterns are broken? :open_mouth:

Comment (erikquinn) [reply src/assets/scripts/client/trafficGenerator/SpawnPatternModel.js]: probably?

**Comment by felixscheffer** [comment]
I have no idea how to do that

**Comment by erikquinn** [comment]
lol ok no worries. i'll check it out and give them a try.

**Comment by erikquinn** [comment]
Definitely doesn't work right. Neither here nor there though, for the purposes of this PR. We do have #156 for their brokenness.

File: src/assets/scripts/client/trafficGenerator/SpawnPatternModel.js (hunk 1 above)
Comment (erikquinn) [reply src/assets/scripts/client/trafficGenerator/SpawnPatternModel.js]: broken indeed. We have #156 for this.

**Pull request merged**